gapic/api/create_new_config.sh

```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
+
+# TODO(jcanizales): What license header to use?
+
+# TODO(jcanizales): Make sure output file doesn't exist.
+
+# Example usage:
+#
+# ./create_new_config.sh clouddebugger v2 devtools/
+#
+# Dependencies need to be added manually after executing.
+#
+# Args:
+# $1 API name (e.g. pubsub)
+# $2 API version (e.g. v1)
+# $3 Path within googleapis/google/ (e.g. devtools/). Without the API name.
+
+API_NAME=$1
```

Comment:
All generation should be driven by artman. We shouldn't have any scripts in googleapis.